pkg/convert/apkbuild/apkbuild.go

```diff
@@ -328,6 +331,7 @@ func (c Context) buildFetchStep(ctx context.Context, converter ApkConvertor) err
 }
 
 // maps APKBUILD values to mconvert
+// Question: Should not we return an error here for better error handling?
 func (c ApkConvertor) mapconvert() {
```

Comment:
Should we return error here for better error handling now that we detect pipelines?